Services/CO.CDP.OrganisationInformation.Persistence/Migrations/20241003131026_QuestionSortOrder.cs

```diff
@@ -0,0 +1,66 @@
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace CO.CDP.OrganisationInformation.Persistence.Migrations
+{
+    /// <inheritdoc />
+    public partial class QuestionSortOrder : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<int>(
+                name: "sort_order",
+                table: "form_questions",
+                type: "integer",
+                nullable: false,
+                defaultValue: 0);
+
+            migrationBuilder.Sql($@"
+                DO $$
+                BEGIN
+                    UPDATE form_questions SET sort_order = 1 WHERE name='_FinancialInformation04';
+                    UPDATE form_questions SET sort_order = 2 WHERE name='_FinancialInformation03';
+                    UPDATE form_questions SET sort_order = 3 WHERE name='_FinancialInformation02';
+                    UPDATE form_questions SET sort_order = 4 WHERE name='_FinancialInformation01';
+                    UPDATE form_questions SET sort_order = 5 WHERE name='_FinancialInformation05';
+
+                    UPDATE form_questions SET sort_order = 1 WHERE name='_Exclusion07';
```

Comment:
Some exclusion questions are missing. This is the full set. You may need to update from main and run your local migrations to get all the new ones.
![image](https://github.com/user-attachments/assets/5442f97e-42c7-4e88-8526-4e5c716bb181)